build.py

```diff
@@ -639,7 +643,7 @@ def enumerate_hdrpr_data(bin_dir):
         assert plugin_dir.exists()
         for f in plugin_dir.glob("**/*"):
             rel_path = f.relative_to(plugin_dir.parent)
-            if any(p in rel_path.parts for p in ("hdRpr", "rprUsd", 'rprUsdMetadata')):
+            if any(p in rel_path.parts and f.name not in ("README.md", ".git", ".gitattributes") for p in ("hdRpr", "rprUsd", 'rprUsdMetadata')):
```

Comment:
This code changes are unreadable, please simplify.
Add `if f.name in ("README.md", ".git", ".gitattributes"): continue` and comment where it could happen